src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -6,6 +6,7 @@
 using System.Diagnostics;
 using System.Globalization;
 using System.Numerics;
+using System.Runtime.InteropServices.WindowsRuntime;
```

Comment:
VS likes adding these lines for some weird reason. I removed it in a later PR.

Reply:
if we are not building UWP any more, we shouldn't use WindowsRuntime I guess. You should punish your VS for doing naughty stuff :-)